.github/workflows/build-and-push.yml

```diff
@@ -71,7 +86,7 @@ jobs:
     if: inputs.push
     strategy:
       matrix:
-        env: [dev, demo]
+        env: [ dev, demo ]
```

Comment:
Your yaml linter is very agressive!

Reply:
I don't even know where it's come from 😅